setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import find_packages, setup
+
+import yolapy
+
+
+setup(
+    name='yolapy',
+    version=yolapy.__version__,
```

Comment:
Ah, right. "Get the version number via code/shell" is enough to convince me.

Comment added in https://github.com/yola/yolapy/commit/33abee9834536703b09057502d392cff75ffa36e